index.js

```diff
@@ -23,7 +23,7 @@ function create (opts) {
   var misc = subleveldown(db, 'misc', {valueEncoding: 'binary'})
   var keys = subleveldown(db, 'added-keys', {valueEncoding: 'binary'})
   var noContent = subleveldown(db, 'no-content', {valueEncoding: 'binary'})
-  var core = hypercore(db)
+  var core = hypercore(storage)
```

Comment:
replicate() depends upon the `core` object, does that mean the replicate function will need to return a generic stream and pipe through the requested hypercore keys?